packages/core/src/util/excludedFromInstrumentation.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
This file contains the main logic for excluding process from auto instrumentation.